Cubical/Categories/Displayed/Functor.agda

```diff
@@ -157,3 +172,56 @@ module _ {C : Category ℓC ℓC'} {D : Category ℓD ℓD'}
   ∫F .F-hom (_ , fᴰ) = _ , Fᴰ.F-homᴰ fᴰ
   ∫F .F-id = ΣPathP (_ , Fᴰ.F-idᴰ)
   ∫F .F-seq _ _ = ΣPathP (_ , (Fᴰ.F-seqᴰ _ _))
+
+-- Projections out of the total category
```

Comment:
This should be in a different file. I think we should move `∫C` to `Displayed.TotalCategory` or `Displayed.TotalCategory.Base` and this can be in that or `Displayed.TotalCategory.Properties`